Tests/Functional/ClangModuleTests.swift

```diff
@@ -97,6 +97,13 @@ class TestClangModulesTestCase: XCTestCase {
             XCTAssertFileExists(prefix, ".build", "debug", "Baz")
         }
     }
+
+    func testDynamicLookup() {
```

Comment:
Please include a comment here (can land after merge) explaining what the test is actually trying to test -- it isn't testing dynamic lookup, it is trying to test that the flags are forwarded to Clang.

One reason I keep harping on this is because eventually I want to refactor our tests to be on a more efficient infrastructure, and it is very hard to do when it isn't clear why a test was added. Hope that makes sense!